website/src/components/Messages/MessageTableEntry.tsx

```diff
@@ -124,6 +124,7 @@ export function MessageTableEntry({ message, enabled, highlight, showAuthorBadge
         onClick={goToMessage}
         cursor={enabled ? "pointer" : undefined}
         style={{ position: "relative" }}
+        overflow="hidden"
```

Comment:
This fix an overflow bug on the dev instance [https://web.dev.open-assistant.io/messages](https://web.dev.open-assistant.io/messages)